src/Rx.DOM.ts

```diff
@@ -1,167 +1,22 @@
-/* tslint:disable:no-unused-variable */
-// Subject imported before Observable to bypass circular dependency issue since
-// Subject extends Observable and Observable references Subject in it's
-// definition
-import {Subject} from './Subject';
-/* tslint:enable:no-unused-variable */
-import {Observable} from './Observable';
-
-// statics
-/* tslint:disable:no-use-before-declare */
-import './add/observable/combineLatest';
-import './add/observable/concat';
-import './add/observable/merge';
-import './add/observable/bindCallback';
-import './add/observable/defer';
-import './add/observable/empty';
-import './add/observable/forkJoin';
-import './add/observable/from';
-import './add/observable/fromArray';
-import './add/observable/fromEvent';
-import './add/observable/fromEventPattern';
-import './add/observable/fromPromise';
-import './add/observable/interval';
-import './add/observable/never';
-import './add/observable/range';
-import './add/observable/throw';
-import './add/observable/timer';
-import './add/observable/zip';
+export * from './Rx';
```

Comment:
Not 100% sure, but I think this might need to be `import './Rx'` now, since the augmentations are treated as a side effect. They recently changed what gets emitted from imports/exports.